testsuite/integration/microprofile-tck/config/src/test/resources/arquillian-bootable.xml

```diff
@@ -9,17 +9,12 @@
             <property name="installDir">${install.dir}</property>
             <property name="jarFile">${bootable.jar}</property>
 
-            <property name="javaVmArguments">${server.jvm.args}</property>
-            <property name="jbossArguments">${jboss.args}</property>
-            <!-- -Djboss.inst is not necessarily needed, only in case the test case needs path to the instance it runs in.
-                 In the future, Arquillian should capable of injecting it into @ArquillianResource File or such. -->
+            <property name="javaVmArguments">${microprofile.jvm.args}</property>
+            <property name="managementAddress">127.0.0.1</property>
+            <property name="managementPort">9990</property>
```

Comment:
Is there a reason why `${as.managementPort:9990}` expression is not being used here?

Reply:
See previous comment.